static/app/views/alerts/rules/metric/details/sidebar.tsx

```diff
@@ -260,6 +262,30 @@ export function MetricDetailsSidebar({
               teamActor ? <ActorAvatar actor={teamActor} size={24} /> : t('Unassigned')
             }
           />
+          {rule.detectionType === 'dynamic' && (
+            <KeyValueTableRow
+              keyName={t('Sensitivity')}
+              value={
+                rule.sensitivity
+                  ? rule.sensitivity.charAt(0).toUpperCase() + rule.sensitivity.slice(1)
+                  : ''
+              } // NOTE: if the rule is dynamic, then there must be a sensitivity
+            />
+          )}
+          {rule.detectionType === 'dynamic' && (
+            <KeyValueTableRow
+              keyName={t('Direction')}
+              value={
+                rule.thresholdType === AlertRuleThresholdType.ABOVE ? (
+                  <OverflowTableValue>{'Above threshold'}</OverflowTableValue>
```

Comment:
nit: I guess the `<OverflowTableValue>` can probably go on the outside of the ternaries?
then we could maybe have a switch case for the strings?